eng/clipack/Aspire.Cli.linux-x64.csproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <CliRuntime>linux-x64</CliRuntime>
     <CliPlatformType>Unix</CliPlatformType>
+    <PublishNativeAot>true</PublishNativeAot>
```

Comment:
You shouldn't need to do this. Just building on a linux machine will set this to `true`.